subgraphs/isolated-pools/src/operations/update.ts

```diff
@@ -193,7 +194,13 @@ export const updateMarket = (
   // Must convert to BigDecimal, and remove 10^18 that is used for Exp in Venus Solidity
   market.borrowRateMantissa = marketContract.borrowRatePerBlock();
 
-  market.supplyRateMantissa = marketContract.supplyRatePerBlock();
+  // calling supplyRatePerBlock can fail due to external reasons, so we fall back to 0 in case of an error
+  const supplyRatePerBlockResult = marketContract.try_supplyRatePerBlock();
+  if (supplyRatePerBlockResult.reverted) {
+    market.supplyRateMantissa = zeroBigInt32;
+  } else {
+    market.supplyRateMantissa = supplyRatePerBlockResult.value;
+  }
```

Comment:
We can wrap this in a function for example
```suggestion
    market.supplyRateMantissa = fallBackToZeroOnRevert(supplyRatePerBlockResult);
```